apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/CarrierClaimTransformerV2.java

```diff
@@ -242,6 +244,13 @@ private static ExplanationOfBenefit transformClaim(CarrierClaim claimGroup) {
           claimGroup.getHcpcsYearCode(),
           Arrays.asList(line.getHcpcsInitialModifierCode(), line.getHcpcsSecondModifierCode()));
 
+      // tax num should be as a extension
```

Comment:
I'm double checking on this.  The mapping documents for Carrier and DME still say "Not Mapping this field" so I'd like to get Jack to update before we move forward with this.

Reply:
i agree

Reply:
Mapping sheet has been updated.  This is all good.  Resolving.